### PR TITLE
Add sponsors carousel below the AAM logo

### LINKED ISSUE #48: Sponsor scroll on the homepage
Attn: @punit121 and @vinaychandil

### PR DESCRIPTION
**Pull request by punit121**
FIXES #48

### CODE REVIEW

File: index.php [hunk 1]
```diff
@@ -54,4 +185,4 @@
 <script src="js/typed.min.js"></script>
 <script src="js/main.js"></script>
 
-</html>
+</html>
```
Comment (icyflame) [inline index.php]: @punit121 

Add a newline at the end of this file.

File: index.php [hunk 2]
```diff
@@ -13,7 +81,70 @@
   <div class="container">
     <div class="col-md-6" id="verticalLine">
       <img src="img/logo-square.jpg" alt="" class="logo" />
+   
+  
+   
+<div id="myCarousel" class="carousel carousel-fade" data-ride="carousel"><h2 align="center"><b>Sponsored by</b></h2>
+  <!-- Indicators
```
Comment (icyflame) [inline index.php]: @punit121 

Remove all of the commented code.

File: index.php [hunk 3]
```diff
@@ -4,6 +4,74 @@
 <head>
   <title>13th Annual Alumni Meet &middot; Home</title>
   <?php include 'topHead.php'; ?>
+ <style>
+ .logo{margin:15px;}
+ <!--#myCarousel{display:box;width:500px;height:175px;;margin-top:-75px;margin-left:-40px;}-->
+ .item{margin-left:155px;}
+ 
+.carousel-fade {
```
Comment (icyflame) [inline index.php]: @punit121 

I am not sure what this CSS is doing here. Please move it to a new file `css/carousel.css` and include that file here.

**Comment by icyflame** [comment]
@punit121 The image for ULA seems to be skewed. Please remove it from the images, if we don't have a square image for that sponsor.

**Comment by icyflame** [comment]
@punit121 
1. Fix ULA image or remove it if it is not square.
2. Rename the folder in which the images are there to `homepage-sponsor-carousel`. Make required changes in the `index.php` file as well.

**Comment by punit121** [comment]
changes done

**Comment by icyflame** [comment]
@punit121 

**Do not change `img/spons/2015/cosponsor.png`, just change the image that has to be used inside `img/spons/sponsors/cosponsor.png`.**

**Comment by punit121** [comment]
changes made

**Comment by icyflame** [comment]
Solved in 4b3aab7174a3580cbaac8b17f2aa37786fe915a7

**Pull request closed without merging**